packages/android_intent/android/src/main/java/io/flutter/plugins/android_intent/AndroidIntentPlugin.java

```diff
@@ -0,0 +1,85 @@
+package io.flutter.plugins.android_intent;
```

Comment:
There's a distinction between files that we've written, and files that are part of the developer's app. We want to never put a license in the developer's app (hence why the template doesn't have one).